circle.yml

```diff
@@ -1,4 +1,6 @@
 machine:
+  node:
+    version: 6.7.0
```

Comment:
I'm concerned about this change because it would mean we'd have to update this in two places when we change node version, which would likely not happen. I can try and look into the build breaking.